src/main/resources/public/src/views/CloneConnector.vue

```diff
@@ -0,0 +1,119 @@
+                    :serializer="displayingResultOfOrganizationSearch"
+                    :append="organizationSelected.siret"
+                    @hit="organizationSelected = $event"    
+                />
+            </div>
+            <div v-if="duplicateErrorMessage !== ''" class="form-group row error-message">{{ duplicateErrorMessage }}</div>
+            <input type="button" value="Create" :disabled="disabled" @click="cloneConnector"/>
+            <input type="button" value="Cancel" @click="backToConnectorsManagament"/>
+        </form>
+    </div>
+</template>
+
+<script>
+
+import VueBootstrapTypeahead from 'vue-bootstrap-typeahead'
+import axios from 'axios'
+import debounce from 'lodash/debounce'
+
+export default {
+    name: 'cloneConnector',
+    components: {
+        VueBootstrapTypeahead
+    },
+    data() {
+        return {
+            selectedConnector: {
+                id: '',
+                applicationName: '',
+                baseUrl: '',
+                organizationSiret: ''
+            },
+            organizations: [],
+            organizationSearch: '',
+            organizationSelected: {},
+            errors: [],
+            duplicateErrorMessage: ''
+        }
+    },
+    watch: {
+        organizationSearch: debounce(function(name) { this.getOrganizations(name) }, 500),
+        organizationSelected: {
+            handler (val, oldVal){
+                this.selectedConnector.organizationSiret = val.siret
+                this.$refs.neworganizationclaimer.$data.inputValue = val.denominationUniteLegale
+            },
+            deep: true
+        }
+    },
+    computed: {
+        disabled (){
+            return Object.keys(this.organizationSelected).length === 0
+        }
+    },
+    created (){
+        this.selectedConnector = {
+            id: this.$route.params.id,
+            applicationName: this.$route.params.appName,
+            baseUrl: ''
+        }
+    },
+    methods: {
+        getOrganizations (name){
+            if(this.organizationSearch === '') this.organizationSelected = {}
+            axios.get("/dc/organizations", {params: {name: name}})
+            .then(response => {
+                this.organizations = response.data
+            })
+            .catch(e => {
+                this.errors.push(e)
+            })
+        },
+        displayingResultOfOrganizationSearch(organization){
+            return `${organization.denominationUniteLegale}, ${organization.siret}`
+        },
+        cloneConnector (){
+            axios.post(`/configuration/connectors/${this.selectedConnector.id}/clone`, this.selectedConnector)
+            .then(response => {
+                this.$router.push({name: 'connectors'})
+            })
+            .catch(e => {
+                this.errors.push(e)
+                this.duplicateErrorMessage = ` ${e}: Connector already exists for application ${this.selectedConnector.applicationName} 
```

Comment:
For the moment we don't manage error message

Reply:
Removed in last commit. But now if the connector already exist it will only be visible in the browser console as a request resulting in HttpStatus 409